gbmodel/model_sqlalchemy.py

```diff
@@ -13,7 +14,7 @@ def getMaxTeamID(self):
             return 1
         else:
             return max_id[0] + 1
-
+            
```

Comment:
Extra spacing

Reply:
Not sure about this -- looks like it matches the others?

Reply:
Not important either way